### PR TITLE
[#664] update README test running instructions and remove Python2 info

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

**Comment by trel** [comment]
looking good - i've got more tweaks in the works.

File: README.md [hunk 1]
```diff
@@ -255,10 +239,9 @@ closed out by the session object's cleanup() method.
 
 Starting with PRC Release 0.9.0, code has been included in the session
 object's `__del__` method to call cleanup(), properly closing out
-network connections. However, `__del__` cannot be relied to run under
-all circumstances (Python2 being more problematic), so an alternative
-may be to call session.cleanup() on any session variable which might not
-be used again.
+network connections. However, `__del__`  is not guaranteed to run as
+expected, so an alternative may be to call session.cleanup()
+on any session variable which might not be used again.
```
Comment (alanking) [inline README.md:244]: Maybe a little firmer?
```suggestion
on any session variable which will not be used again.
```

**Review by alanking — COMMENTED** [review]
Seems good. Just one wording suggestion from me.

File: README.md (hunk 1 above)
Comment (trel) [reply README.md]: agreed

**Comment by trel** [comment]
did a full readthrough for consistency and modernization.

probably can get (re)organized a bit more later.   there's a lot.   but this feels good enough for a .0.

**Review by alanking — COMMENTED** [review]
New words look good. Anything else left for this? Time to squash?

**Comment by trel** [comment]
i will squash them.

**Review by alanking — APPROVED** [review]
Sweet. # it

**Comment by trel** [comment]
#'d

**Pull request merged**